src/interfaces/webhook-config-interface.ts

```diff
@@ -0,0 +1,4 @@
+export default interface IWebhookConfig {
```

Comment:
this looks more default than just webhook-related, can be called something like IConfigItem